spec/README.adoc

```diff
@@ -0,0 +1,53 @@
+= Jakarta Data Specification
+
+This module contains AsciiDoc sources and configuration to generate Jakarta Data documentation in HTML and PDF formats for both Apache License 2 and Jakarta Eclipse Foundation Specification Process.
+
+== Generating the Documentation
+
+=== Prerequisites:
+
+* JDK 11+
+* Maven 3.0.3+
```

Comment:
But
```
[ERROR] Failed to execute goal org.asciidoctor:asciidoctor-maven-plugin:2.1.0:process-asciidoc (generate-pdf-doc) on project jakarta-data-spec: The plugin org.asciidoctor:asciidoctor-maven-plugin:2.1.0 requires Maven version 3.0.5
```

Reply:
OK, fair enough.  Despite the fact that Maven 3.0.3 isn't even available on SDKMan, for example, it's a good idea to test the documentation.  Since I adapted this README.doc file from Jakarta CDI, I assume that a lot of the specifications would need to update this Maven version.